src/main/java/com/exadel/discountwebapp/statistics/controller/StatisticsController.java

```diff
@@ -27,21 +27,21 @@
     @GetMapping
     public SummaryStatisticsDTO getStatistics(@RequestParam(value = "dateFrom") @DateTimeFormat(pattern = "dd.MM.yyyy", iso = DateTimeFormat.ISO.DATE) LocalDate dateFrom,
                                               @RequestParam(value = "dateTo") @DateTimeFormat(pattern = "dd.MM.yyyy", iso = DateTimeFormat.ISO.DATE) LocalDate dateTo) {
-        return statisticsService.getStats(dateFrom.atStartOfDay(), dateTo.atStartOfDay());
+        return statisticsService.getStats(dateFrom.atStartOfDay(), dateTo.atStartOfDay().plusHours(24));
```

Comment:
why do you add 24 hours?

Reply:
it's for set range date during one day, for example, dateFrom=16.07.2021&dateTo=16.07.2021, where dateFrom -start of the day and dateTo - end of the day